### ISSUE TITLE
Making Information Easier to Understand - AI Bug Bash

### ISSUE THREAD

**Issue by curriee11** [microsoft/studentambassadors#143]
## Describe the bug
simplify the content to make it more accessible for students as hard information is generally avoided and skipped by students

## To Reproduce
[Flask Fundamentals](https://learn.microsoft.com/en-us/training/modules/python-flask-build-ai-web-app/2-flask-fundamentals)
It could be made easier 
By following steps it could become easier:
Flask is a web framework that allows you to build web applications. It uses a simple approach, called a micro-framework, which means it only includes the basic features required to build a web application.

To build a web application, you need to understand three main concepts: routing, methods, and templates.

Routing is the process of mapping URLs to functions. This means when a user types in a specific URL, Flask knows which function to call to handle that URL.

Methods are used to provide additional context about the user's request. The two most common methods are GET and POST. GET is used when the user wants to retrieve information, and POST is used when the user needs to send information to the server.

Templates are used to make the web pages dynamic. They allow you to programmatically insert information into an HTML page. Flask uses a templating engine called Jinja, which uses placeholders to indicate where the dynamic information should be inserted.

In summary, Flask is a simple web framework that uses routing, methods, and templates to build web applications. By understanding these concepts, you can build web applications with Flask.

#### 🎓 Add a tag to this issue for your current education role: **student**, **Student Ambassador** 
**student Ambassador** , **AI bug bash**

***

**Comment by Muhammad-Qasim11** [comment]
I think the content in the modules is clear and easy. It is obvious that only those people will read these content who are from tech industry and those who are working in tech fields should have the skill of how to read a tech related content and I know that technical writing courses and other content writing courses are taught in every university which prepares you how to write and read a tech related document.
More over when Microsoft experts writes these modules then they focus on how to deliver more knowledge in few lines because if they write details of each and every topic they the readers will not be able to focus on their intended learning topic and mostly students get bored when they see a huge web page full of written content.
You should have the skill to research the topic by yourself using Google or chatgpt or any other forums. If don't understand any part of the content then you can post your queries in Q & A section of Microsoft learn websites. And if you are MLSA then you can also use MS teams channels to ask questions from your fellow ambassadors. 
If you are MLSA, then you can login to your visual studio subscription where you will find free subscription for different Q & A websites of Microsoft. Below are the free subscriptions available in your visual studio subscription:
Azure community (forum support)
Microsoft Q & A (forum support)
Microsoft tech community (online user support)
Developer community (forum support)
Concierge chat (24/7)
These above websites are very useful to find any information regarding Microsoft technologies and if your facing any issues in working with Microsoft technologies then you can ask your questions in these support forums.
So the modules designed are clear and easy to understand,  if you read them with full focus. For any other issues specific to you then you can use the above support forums.
I hope this make sense and for any other support I am available.
Regards 
Muhammad Qasim 
Beta MLSA
